build.gradle

```diff
@@ -1,70 +1,58 @@
 buildscript {
   repositories {
-    mavenCentral()
+    jcenter()
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:1.3.1'
-    classpath 'com.f2prateek.checkstyle:checkstyle:1.0.1'
+    classpath 'com.android.tools.build:gradle:2.3.3'
+    classpath 'com.f2prateek.javafmt:javafmt:0.1.6'
   }
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'checkstyle'
-apply plugin: 'com.f2prateek.checkstyle'
+apply plugin: 'com.f2prateek.javafmt'
 
 android {
-  compileSdkVersion 23
-  buildToolsVersion '23.0.1'
+  compileSdkVersion 26
+  buildToolsVersion '25.0.3'
 
   defaultConfig {
     minSdkVersion 14
-    targetSdkVersion 23
-    compileSdkVersion 23
+    targetSdkVersion 26
+    compileSdkVersion 26
   }
 
   compileOptions {
     sourceCompatibility JavaVersion.VERSION_1_7
     targetCompatibility JavaVersion.VERSION_1_7
   }
-
-  lintOptions {
-    // Okio/OkHTTP.
-    disable 'InvalidPackage'
-  }
-}
-
-checkstyle {
-  configFile rootProject.file('gradle/checkstyle.xml')
 }
 
 dependencies {
   repositories {
+    jcenter()
     mavenCentral()
   }
 
-  provided 'com.segment.analytics.android:analytics:4.0.0'
+  compile 'com.optimizely.ab:android-sdk:+'
```

Comment:
For Android we have refrained from using `+` - best practice dictates to specify version numbers as `+` can lead to unpredictable builds - not sure if we have a strong opinion here